index.html

```diff
@@ -1010,6 +1011,11 @@ <h3>Processing Errors</h3>
         <li>
 The `detail` value SHOULD provide a longer human-readable string for the error.
         </li>
+        <li>
+All human-readable strings SHOULD support internationalization using
+the `Accept-Language` HTTP header field as described in Section 1 of
+[[[RFC9457]]].
```

Comment:
Several problems here.

- I wouldn't say "support internationalization" because that doesn't mean very much. Probably this should reference localization.
- Language negotiation is not really defined by RFC9457. This is not the RFCs fault: it is a deliberate choice not to over-define what can be a complex process.

I would suggest:

```suggestion
All human-readable strings SHOULD be localized as described in Section 1 of
[[[RFC9457]]], using [=language negotiation=] on the `Accept-Language` HTTP
header field to select the most appropriate resources.
```

The slightly tortured syntax lets me work in the reference to I18N Glossary's term `language negotiation`, which in turn points to both LTLI (more details if someone wants them) and to the most common language negotiation schemes (BCP47's and JavaScript's)

Reply:
Thanks for the correction, as always @aphillips, merging your changes in.